### ISSUE TITLE
Feature: Support VMSS and low priority

### ISSUE THREAD

**Issue by Gaploid** [Azure/AKS#290]
Hi,
It will be awesome to add support vmss and low priority vms with AKS.

**Comment by ahelland** [comment]
Possibly handled by k8s natively instead of AKS?
From https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md#azure
"Kubernetes 1.10 includes alpha Azure support for cluster-autoscaler, as well as support for Azure Virtual Machine Scale Sets."

**Comment by seanmck** [comment]
Please add this as a feature request in our feedback forum so we can prioritize it:

https://aka.ms/aks/feedback

**Comment by FelschR** [comment]
I've just went ahead and created one:
https://feedback.azure.com/forums/914020-azure-kubernetes-service-aks/suggestions/35077432-low-priority-vms-and-vmss

**Comment by ilooner** [comment]
@seanmck Are there any plans to implement this feature?

**Comment by DovAmir** [comment]
why is this closed? is low priority suported in AKS?

**Comment by iftachsc** [comment]
+1

**Comment by yevegnytr** [comment]
+1

**Comment by CoRfr** [comment]
I just spent almost a week on the phone with Azure helpdesk, and while attaching disks to VMSS Low priority used to be supported (I used it without any issues between October 2018 - March 16th 2019), and it appears not to work at the moment.

API works fine on Regular priority VMSS (PUT call on Microsoft.Compute/virtualMachineScaleSets/<VMSS>/virtualMachines/<instanceId>), but it fails on Low priority with:
```
{"error":{"code":"OperationNotAllowed","message":""}}
```
there is also a `Conflict (HTTP Status Code: 409)` thrown in the reply.

I imagine that it would be problematic for the AKS implementation.
